scripts/initialize_andi_req_data/create_org_one.json

```diff
@@ -8,4 +8,4 @@
   "orgName": "urbanos_demo_org",
   "orgTitle": "UrbanOS Demo Org",
   "version": "0.1"
-}
+}
```

Comment:
mmm, yes, very good change here /j (I figure this is a formatting change, but funny nonetheless)